barretenberg/cpp/src/barretenberg/ultra_honk/decider_prover.cpp

```diff
@@ -17,7 +17,7 @@ DeciderProver_<Flavor>::DeciderProver_(const std::shared_ptr<Instance>& inst,
                                        const std::shared_ptr<Transcript>& transcript)
     : accumulator(std::move(inst))
     , transcript(transcript)
-    , commitment_key(inst->proving_key->commitment_key)
+    , commitment_key(accumulator->proving_key->commitment_key)
```

Comment:
probably not meaningful, but I wasn't sure

Reply:
actually, shouldn't inst be nullified by the std::move on line 18?

Reply:
std::move doesn't nullify the object being moved it just facilities moving rather than copying resources, your change makes sense but the previous code wasn't incorrect